src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -1087,6 +1087,9 @@
 
     /** Table head tag name. */
     public static final int THEAD_HTML_TAG_NAME = JavadocParser.THEAD_HTML_TAG_NAME;
+
+    /** `optgroup` tag name. (JavadocLexer token) */
```

Comment:
Why do we need the text ` (JavadocLexer token)`? None of the others above it have it.

Reply:
Removed it.